spec/cli_spec.rb

```diff
@@ -46,6 +46,33 @@
         end
 
       end
+
+      context "when git is not installed" do
+        before do
+          expect(Bundler::Audit::Database).to receive(:update!).and_return(nil)
```

Comment:
The specs were all updated recently. Please see about conforming this and any other code to the recent style changes. For example, probably can use described_class here?

Reply:
`described_class` is `Bundler::Audit::CLI` in this case, so cannot be used
I do not see inconsistencies with surrounding code.

Reply:
Thanks! I was reviewing from my phone, so I couldn't see full file context. :-)